apps/web/build/fetchFavicon.ts

```diff
@@ -16,23 +15,12 @@ const fetchFavicon = async (data: { [key: string]: Array<ConfigItem> }) => {
     return;
   }
 
-  try {
-    // eslint-disable-next-line no-console
-    console.log('Fetching favicon from:', faviconUrl.value);
+  // eslint-disable-next-line no-console
+  console.log('Fetching favicon from:', faviconUrl.value);
```

Comment:
To Do: This needs to be removed

Reply:
I want to keep this, so that the user gets a progress indicator in the terminal. We have the no-console rule to prevent logs in the browser console, but this is a different use case in my opinion.